inc/cleantalk-public-integrations.php

```diff
@@ -1654,18 +1654,9 @@ function ct_check_registration_erros($errors, $_sanitized_user_login = null, $_u
  */
 function apbct_user_register($user_id)
 {
-    global $apbct_cookie_request_id_label, $apbct_cookie_request_id;
-
-    if ( ! empty($apbct_cookie_request_id) ) {
-        update_user_meta($user_id, 'ct_hash', $apbct_cookie_request_id);
-
-        return;
-    }
-
-    if ( isset($_COOKIE[$apbct_cookie_request_id_label]) ) {
-        if ( update_user_meta($user_id, 'ct_hash', $_COOKIE[$apbct_cookie_request_id_label]) ) {
-            Cookie::set($apbct_cookie_request_id_label, '0', 1, '/');
-        }
+    $hash = ct_hash();
```

Comment:
why call ct_hash with an empty parameter?

Reply:
Using a static variable in the function.